include/NovelRT/Input/KeyCode.h

```diff
@@ -8,6 +8,9 @@
 #endif
 
 namespace NovelRT::Input {
+  /**
+   * A set of key codes, with their OpenGL values.
```

Comment:
Incorrect. These are the values assigned by GLFW, but that's an implementation detail and doesn't need to be shared.

Reply:
Oh literally just read "GL" and though that was OpenGL, i'm just going to remove that